Dockerfile

```diff
@@ -15,12 +17,20 @@ USER app
 COPY --chown=app ./requirements.txt /app/requirements.txt
 RUN pip install -r requirements.txt
 
+RUN echo "Node: " && node -v
+RUN echo "NPM: " && npm -v
+
+COPY package*.json ./
+RUN npm install
```

Comment:
Is this enough? Does the `django-gulp` logic hook right into the existing `collectstatic` step?

Reply:
cc/ @moz-astults 

Quick context for what this patch resolves. We were getting [this error](https://app.circleci.com/pipelines/github/mozilla/fx-private-relay/1640/workflows/62add57c-67ef-42ab-a409-60d71c191da8/jobs/3102) from Circle CI: 

```bash
/bin/sh: 1: gulp: not found
CommandError: Command 'gulp build --cwd /app --production' returned non-zero exit status 127.
The command '/bin/sh -c python manage.py collectstatic --no-input -v 2' returned a non-zero code: 1

Exited with code exit status 1
```

This lead me down the rabbit hole of making sure node/npm is included in the Container file.

Reply:
Also we can totally remove the `echo` lines. That was just for debugging.

Reply:
> Does the django-gulp logic hook right into the existing collectstatic step?

@groovecoder Yeah. This resolves the original error linked above.